README.md

```diff
@@ -19,7 +19,9 @@ We use the same general guidelines for introducing a new DIP as specified in [BI
 Number | Layer | Title | Owner | Type | Status
 --- | --- | --- | --- | --- | ---
 [1](dip-0001.md) | Consensus | Initial Scaling of the Network | Darren Tapp | Standard | Final
-
```

Comment:
Two blank lines have been used to separate text body from new h2 headers thus far in this doc. Did you remove this one intentionally?

Reply:
Nope. Fixed!